sync-todo/v2/client/maui/RealmTodo/ViewModels/ItemsViewModel.cs

```diff
@@ -47,16 +47,8 @@ public async Task Logout()
         [RelayCommand]
         public async Task AddItem()
         {
-            var promptResult = await DialogService.ShowPromptAsync("New item");
-
-            if (!string.IsNullOrEmpty(promptResult))
-            {
-                await realm.WriteAsync(() =>
-                {
-                    var item = new Item() { OwnerId = currentUserId, IsComplete = false, Summary = promptResult };
-                    realm.Add(item);
-                });
-            }
+            var item = new Dictionary<string, object>() { { "item", new Item() } };
```

Comment:
This can't be called `item`, it's confusing. `navigationParameters` makes more sense to call it `itemParameter` like in the other method